api/lib/application/organizations/index.js

```diff
@@ -134,7 +134,7 @@ exports.register = async (server) => {
           maxBytes: 1048576 * 10, // 10MB
           parse: 'gunzip',
         },
-        handler: organisationController.importStudentsFromSIECLE,
+        handler: organisationController.importSchoolingRegistrationsFromSIECLE,
         notes: [
           '- **Cette route est restreinte aux utilisateurs authentifiés et responsables de l\'organisation**\n' +
           '- Elle permet d\'importer des élèves en masse depuis un fichier au format SIECLE\n' +
```

Comment:
```suggestion
          '- Elle permet d\'importer des inscription d'élèves, en masse, à partir d'un export SI SIECLE\n' +
```

Reply:
+1 sans oublier le S à "inscriptions" !

Reply:
Je préfère la fin de la première version car on ne sait pas (et on s'en fiche un peu) de savoir si le fichier au format siècle que l'on reçoit a été exporté ou si il a été maintenu à la main. Du coup j'opterais pour: "Elle permet d'importer des inscriptions d'élèves, en masse, depuis un fichier au format SIECLE". Good for you ?

Reply:
Qu'ajoute "en masse" au verbe "importer" ?